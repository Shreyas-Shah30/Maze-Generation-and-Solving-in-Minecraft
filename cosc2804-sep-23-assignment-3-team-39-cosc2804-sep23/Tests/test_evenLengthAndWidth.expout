
Welcome to MineCraft MazeRunner!
--------------------------------

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of maze: 
Enter the length and width of maze: 

Length and width should be odd numbers.
Length and width should be odd numbers.

Maze generated successfully
**Printing Maze**
BasePoint: (4024, 65, 4226)
Structure:
xxxxxxx
......x
xxxxx.x
x...x.x
x.x.x.x
x.x...x
xxxxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

The End!