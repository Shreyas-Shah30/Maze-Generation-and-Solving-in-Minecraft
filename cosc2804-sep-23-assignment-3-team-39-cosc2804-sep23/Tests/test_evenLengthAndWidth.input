1
2
4013 64 4203
4 5
5 8
7 7
6
