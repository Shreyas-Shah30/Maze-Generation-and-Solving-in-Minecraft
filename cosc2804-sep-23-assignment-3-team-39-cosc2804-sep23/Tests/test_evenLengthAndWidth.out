
Welcome to MineCraft MazeRunner!
--------------------------------

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of the maze:
Enter the length and width of the maze:
Length and width should be odd numbers.
Maze generated successfully
**Printing Maze**
BasePoint: (4013, 64, 4203)
Structure:
xxxxxxx
......x
xxxxx.x
x...x.x
x.x.x.x
x.x...x
xxxxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 
The End!

